sig-robotics/README.md

```diff
@@ -23,18 +23,15 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
 
 The Chairs of the SIG run operations and processes governing the SIG.
```

Comment:
although it is such an honor to be `Charis` for me, i am not really clear `operations and processes` by here? is there any description else where? if SIG robotics does not have it in concrete yet, it would be really helpful to point the other SIG charters or something else to give overviews.

i am not against this change, but just checking what i am responsible to do so.

Reply:
hi, Tomoya. 

I hope, as the chair, you are aware that we are a SIG in development and committed to finalizing a personalized charter and governance system within this year. And, I want to assure you that your willingness to contribute is highly valued. 

[This](https://github.com/kubeedge/community/blob/master/GOVERNANCE.md)  is a Work in Progress, documenting approximately how we have been operating up to this point.

> The governance model adopted here is heavily influenced by a set of CNCF projects, especially drew reference from [Kubernetes governance](https://github.com/kubernetes/community/blob/master/governance.md).

I understand that the [SIG Robotics charter](https://github.com/kubeedge/community/blob/master/sig-robotics/charter.md) DO NOT have detailed operational guidelines at this time. 

While the charter may not outline specific processes yet, your role as Charis remains central to creating a welcoming and collaborative environment.

Thank you for your dedication to SIG Robotics, and we look forward to your contributions.

Reply:
@JoeyHwong-gk thanks for the information. i am willing to do so, in the meantime if i will try to consider what i can do fo https://github.com/kubeedge/community/tree/master/sig-robotics. thanks!